app-text/multimarkdown/multimarkdown-6.4.0.recipe

```diff
@@ -0,0 +1,82 @@
+SUMMARY="Lightweight markup processor to produce HTML, LaTeX, and more"
+DESCRIPTION="MultiMarkdown, or MMD, is a tool to help turn minimally marked-up \
+plain text into well formatted documents, including HTML, PDF (by way of LaTeX), \
+OPML, or OpenDocument (specifically, Flat OpenDocument or ‘.fodt’, which can \
+in turn be converted into RTF, Microsoft Word, or virtually any other \
+word-processing format)."
+HOMEPAGE="https://fletcherpenney.net/multimarkdown/"
+COPYRIGHT="2016-2018, Fletcher T. Penney"
+LICENSE="multimarkdown"
+REVISION="1"
+SOURCE_URI="https://github.com/fletcher/MultiMarkdown-6/archive/$portVersion.tar.gz"
+CHECKSUM_SHA256="fa9daa089bc0f8bc1f69fd7365778d59210b446ce7008d03b87fb572df8ff5d3"
+SOURCE_DIR="MultiMarkdown-6-$portVersion"
+PATCHES="multimarkdown-$portVersion.patch"
+
+ARCHITECTURES="!x86_gcc2 x86_64"
```

Comment:
Suggestion:
```diff
-ARCHITECTURES="!x86_gcc2 x86_64"
+ARCHITECTURES="!x86_gcc2 ?x86 x86_64"
```
If there is still someone running x86 they could check it out also :)